src/controllers/heartbeat.js

```diff
@@ -8,7 +8,7 @@ const router = express.Router();
 
 router.get('/', (req, res) => {
   logger.silly(`Got ${req.originalUrl}`);
-  res.status(200).json({ msg: 'Success', code: 200 });
+  res.status(200).json({ message: 'Success', code: 200 });
```

Comment:
Could you add to the pr description that you changed the msg key here?

Reply:
done